src/Microsoft.PowerShell.ConsoleHost/WindowsTaskbarJumpList/TaskbarJumpList.cs

```diff
@@ -33,13 +33,12 @@ internal static void CreateRunAsAdministratorJumpList()
             {
                 try
                 {
-                    TaskbarJumpList.CreateElevatedEntry(ConsoleHostStrings.RunAsAdministrator);
+                    CreateElevatedEntry(ConsoleHostStrings.RunAsAdministrator);
                 }
-                catch (Exception exception)
+                catch (Exception)
```

Comment:
Could you please move unrelated changes in another PR?

Reply:
It's related. When running `Start-Process pwsh -Credential` as a non-admin user, the task bar jump list code to throw `AccessDenied` exception, and thus the `Debug.Fail` call will cause a debug build to crash, which brought trouble to the testing.

The `Debug.Fail` is not needed. The `catch (ThreadStartException)` below simply ignore the exception too.